<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.26" />
  <meta name="author" content="Junpeng Zhang">
  <meta name="description" content="Associate Professor of Bioinformatics">

  
  
  
    
  
  
    
    
    <link rel="stylesheet" href="/css/highlight.min.css">
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.0/css/academicons.min.css" integrity="sha512-GGGNUPDhnG8LEAEDsjqYIQns+Gu8RBs4j5XGlxl7UfRaZBhCCm5jenJkeJL8uPuOXGqgl8/H1gjlWQDRjd3cUQ==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-105253164-1', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>    
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  <link rel="alternate" href="https://zhangjunpeng411.github.io/index.xml" type="application/rss+xml" title="JUNPENG ZHANG">
  <link rel="feed" href="https://zhangjunpeng411.github.io/index.xml" type="application/rss+xml" title="JUNPENG ZHANG">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="https://zhangjunpeng411.github.io/">

  

  <title>JUNPENG ZHANG</title>

  
  
<script>
window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
ga('create', 'UA-105253164-1', 'auto');
ga('send', 'pageview');
</script>
<script async src='//www.google-analytics.com/analytics.js'></script>

    

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">
<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">JUNPENG ZHANG</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/#about" data-target="#about">
            
            <span>Home</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#publications_selected" data-target="#publications_selected">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#posts" data-target="#posts">
            
            <span>Posts</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#projects" data-target="#projects">
            
            <span>Projects</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#teaching" data-target="#teaching">
            
            <span>Teaching</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#contact" data-target="#contact">
            
            <span>Contact</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>



  
  
  <section id="about" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('https://zhangjunpeng411.github.io/img/Junpeng_Zhang3.png');"></div>
      <meta itemprop="image" content="https://zhangjunpeng411.github.io/img/Junpeng_Zhang3.png">
      

      <div class="portrait-title">
        <h2 itemprop="name">Junpeng Zhang</h2>
        <h3 itemprop="jobTitle">Associate Professor of Bioinformatics</h3>
        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <a href="http://www.dali.edu.cn/" target="_blank" itemprop="url">
            <span itemprop="name">Dali University</span>
          </a>
        </h3>
        
      </div>

      <link itemprop="url" href="https://zhangjunpeng411.github.io/">

      <ul class="social-icon" aria-hidden="true">
        
        
        <li>
          <a itemprop="sameAs" href="mailto:zjp@dali.edu.cn" target="_blank">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="mailto:zhangjunpeng_411@yahoo.com" target="_blank">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://www.researchgate.net/profile/Junpeng_Zhang3" target="_blank">
            <i class="ai ai-researchgate big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://scholar.google.com.au/citations?hl=en&amp;view_op=list_works&amp;gmla=AJsN-F4U77WnwbOmpd6hFe0bTypArKx236B-P5MrDPTZeIAQDqjuOwOUMuAsBTEjb-fI7PcCR7cLckMA5yXZ62wDPH_6a87I2w&amp;user=u5wIFyoAAAAJ" target="_blank">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://github.com/zhangjunpeng411" target="_blank">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="http://orcid.org/0000-0001-6127-9701" target="_blank">
            <i class="ai ai-orcid big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">

    

<h1 id="biography">Biography</h1>

<p>Junpeng Zhang is an associate professor of Binformatics at the school of Engineering, Dali University. His research interests include Bioinformatics, Computational Biology and Data Mining. He is specially interested in identifying miRNA mediated regulatory network.</p>


    <div class="row">

      
      <div class="col-sm-5">
        <h3>Interests</h3>
        <ul class="ul-interests">
          
          <li>Bioinformatics</li>
          
          <li>Computational Biology</li>
          
          <li>Data Mining</li>
          
        </ul>
      </div>
      

      
      <div class="col-sm-7">
        <h3>Education</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">MSc in Control Theory and Control Engineering, 2009</p>
              <p class="institution">Kunming University of Science and Technology</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">BSc in Bio-medical Engineering, 2005</p>
              <p class="institution">Kunming University of Science and Technology</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>

    </div>
  </section>

  
  
  <section id="publications_selected" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Selected Publications</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/mirscoppi/">
        <img src="/img/miRSCoPPI.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/mirscoppi/" itemprop="url">Inferring miRNA sponge co-regulation of protein-protein interactions in human breast cancer</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Background Recent studies have shown that the crosstalk between microRNA (miRNA) sponges plays an important role in human cancers. However, the co-regulation roles of miRNA sponges in protein-protein interactions (PPIs) are still unknown. Results In this study, we propose a multi-step method called miRSCoPPI to infer miRNA sponge co-regulation of PPIs. We focus on investigating breast cancer (BRCA) related miRNA sponge co-regulation, by integrating heterogeneous data, including miRNA, long non-coding RNA (lncRNA) and messenger RNA (mRNA) expression data, experimentally validated miRNA-target interactions, PPIs and lncRNA-target interactions, and the list of breast cancer genes. We find that the inferred BRCA-related miRSCoPPI network is highly connected and scale free. The top 10% hub genes in the BRCA-related miRSCoPPI network have potential biological implications in breast cancer. By utilizing a graph clustering method, we discover 17 BRCA-related miRSCoPPI modules. Through pathway enrichment analysis of the modules, we find that several modules are significantly enriched in pathways associated with breast cancer. Moreover, 10 modules have good performance in classifying breast tumor and normal samples, and can act as module signatures for prognostication. By using putative computationally predicted miRNA-target interactions, we have consistent results with those obtained using experimentally validated miRNA-target interactions, indicating that miRSCoPPI is robust in inferring miRNA sponge co-regulation of PPIs in human breast cancer. Conclusions Taken together, the results demonstrate that miRSCoPPI is a promising tool for inferring BRCA-related miRNA sponge co-regulation of PPIs and it can help with the understanding of the co-regulation roles of miRNA sponges on the PPIs.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Thuc Duy Le, Lin Liu, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        BMC Bioinformatics
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirscoppi/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1672-2">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/316784504_Inferring_miRNA_sponge_co-regulation_of_protein-protein_interactions_in_human_breast_cancer">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/mirsm/">
        <img src="/img/miRSM.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/mirsm/" itemprop="url">Identifying miRNA sponge modules using biclustering and regulatory scores</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Background MicroRNA (miRNA) sponges with multiple tandem miRNA binding sequences can sequester miRNAs from their endogenous target mRNAs. Therefore, miRNA sponge acting as a decoy is extremely important for long-term loss-of-function studies both in vivo and in silico. Recently, a growing number of in silico methods have been used as an effective technique to generate hypotheses for in vivo methods for studying the biological functions and regulatory mechanisms of miRNA sponges. However, most existing in silico methods only focus on studying miRNA sponge interactions or networks in cancer, the module-level properties of miRNA sponges in cancer is still largely unknown. ResultsWe propose a novel in silico method, called miRSM (miRNA Sponge Module) to infer miRNA sponge modules in breast cancer. We apply miRSM to the breast invasive carcinoma (BRCA) dataset provided by The Cancer Genome Altas (TCGA), and make functional validation of the computational results. We discover that most miRNA sponge interactions are module-conserved across two modules, and a minority of miRNA sponge interactions are module-specific, existing only in a single module. Through functional annotation and differential expression analysis, we also find that the modules discovered using miRSM are functional miRNA sponge modules associated with BRCA. Moreover, the module-specific miRNA sponge interactions among miRNA sponge modules may be involved in the progression and development of BRCA. Our experimental results show that miRSM is comparable to the benchmark methods in recovering experimentally confirmed miRNA sponge interactions, and miRSM outperforms the benchmark methods in identifying interactions that are related to breast cancer. Conclusions Altogether, the functional validation results demonstrate that miRSM is a promising method to identify miRNA sponge modules and interactions, and may provide new insights for understanding the roles of miRNA sponges in cancer progression and development.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Thuc Duy Le, Lin Liu, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        BMC Bioinformatics
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirsm/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1467-5">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/315062626_Identifying_miRNA_sponge_modules_using_biclustering_and_regulatory_scores">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/mirsrn/">
        <img src="/img/mirSRN.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/mirsrn/" itemprop="url">Identifying miRNA synergistic regulatory networks in heterogeneous human data via network motifs</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Understanding the synergism of multiple microRNAs (miRNAs) in gene regulation can provide important insights into the mechanisms of complex human diseases caused by miRNA regulation. Therefore, it is important to identify miRNA synergism and study miRNA characteristics in miRNA synergistic regulatory networks. A number of methods have been proposed to identify miRNA synergism. However, most of the methods only use downstream target genes of miRNAs to infer miRNA synergism when miRNAs can also be regulated by upstream transcription factors (TFs) at the transcriptional level. Additionally, most methods are based on statistical associations identified from data without considering the causal nature of gene regulation. In this paper, we present a causality based framework, called mirSRN (miRNA synergistic regulatory network), to infer miRNA synergism in human molecular systems by considering both downstream miRNA targets and upstream TF regulation. We apply the proposed framework to two real world datasets and discover that almost all the top 10 miRNAs with the largest node degree in the mirSRNs are associated with different human diseases, including cancer, and that the mirSRNs are approximately scale-free and small-world networks. We also find that most miRNAs in the networks are frequently synergistic with other miRNAs, and miRNAs related to the same disease are likely to be synergistic and in a cluster linked to a biological function. Synergistic miRNA pairs show higher co-expression level, and may have potential functional relationships indicating collaboration between the miRNAs. Functional validation of the identified synergistic miRNAs demonstrates that these miRNAs cause different kinds of diseases. These results deepen our understanding of the biological meaning of miRNA synergism.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Thuc Duy Le, Lin Liu, Jianfeng He, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        Molecular BioSystems
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirsrn/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://pubs.rsc.org/-/content/articlelanding/2016/mb/c5mb00562k/unauth#!divAbstract">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/284765822_Identifying_miRNA_synergistic_regulatory_network_in_heterogeneous_human_data_via_network_motifs">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/mirlab/">
        <img src="/img/miRLAB.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/mirlab/" itemprop="url">miRLAB: An R Based Dry Lab for Exploring miRNA-mRNA Regulatory Relationships</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        microRNAs (miRNAs) are important gene regulators at post-transcriptional level, and inferring miRNA-mRNA regulatory relationships is a crucial problem. Consequently, several computational methods of predicting miRNA targets have been proposed using expression data with or without sequence based miRNA target information. A typical procedure for applying and evaluating such a method is i) collecting matched miRNA and mRNA expression profiles in a specific condition, e.g. a cancer dataset from The Cancer Genome Atlas (TCGA), ii) applying the new computational method to the selected dataset, iii) validating the predictions against knowledge from literature and third-party databases, and comparing the performance of the method with some existing methods. This procedure is time consuming given the time elapsed when collecting and processing data, repeating the work from existing methods, searching for knowledge from literature and third-party databases to validate the results, and comparing the results from different methods. The time consuming procedure prevents researchers from quickly testing new computational models, analysing new datasets, and selecting suitable methods for assisting with the experiment design. Here, we present an R package, miRLAB, for automating the procedure of inferring and validating miRNA-mRNA regulatory relationships. The package provides a complete set of pipelines for testing new methods and analysing new datasets. miRLAB includes a pipeline to obtain matched miRNA and mRNA expression datasets directly from TCGA, 12 benchmark computational methods for inferring miRNA-mRNA regulatory relationships, the functions for validating the predictions using experimentally validated miRNA target data and miRNA perturbation data, and the tools for comparing the results from different computational methods.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Thuc Duy Le, Junpeng Zhang, Lin Liu, Huawen Liu, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        PLoS ONE
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirlab/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0145386">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/288830678_MiRLAB_An_R_based_dry_lab_for_exploring_miRNA-mRNA_regulatory_relationships">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/coreppi/">
        <img src="/img/CoRePPI.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/coreppi/" itemprop="url">A novel framework for inferring condition-specific TF and miRNA co-regulation of protein-protein interactions</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Recent studies have shown that transcription factors (TFs) and microRNAs (miRNAs), while independently regulate their downstream targets, collaborate with each other to regulate gene expression. However, their synergistic roles in protein-protein interactions (PPIs) remain mostly unknown. In this paper, we present a novel framework (called CoRePPI) for inferring TF and miRNA co-regulation of PPIs. Particularly, CoRePPI is aimed at discovering the co-regulation specific to a condition of interest, by using heterogeneous data, including miRNA and messenger RNA (mRNA) expression profiles, putative miRNA targets, TF targets and PPIs. CoRePPI firstly finds the network motifs indicating the co-regulation of PPIs by TFs and miRNAs in tumor and normal conditions separately. Then by identifying the differential motifs found in one condition but not in the other, it builds the networks consisting of TFs, miRNAs and their co-regulated PPIs specific to different conditions respectively. To validate CoRePPI, we apply it to the Pan-Cancer dataset which includes the expression profiles of 12 cancer types from TCGA. Through network topology analysis, we found that the tumor and normal CoRePPI networks are scale-free. Furthermore, the results of differential and intersected network analysis between the tumor and normal CoRePPI networks suggest that only a small fraction of the regulatory relationships between TFs and miRNAs are conserved in both conditions but they co-regulate different downstream PPIs in tumor and normal conditions; and in different conditions the majority of the regulatory relationships between TFs and miRNAs are different although they may regulate the same PPIs in their respective conditions. The CoRePPI sub-networks constructed for the three types of cancers (breast cancer, lung cancer and ovarian cancer) are all scale-free, and the intersection of these CoRePPI sub-networks can be utilized as the biomarker CoRePPI sub-network of the three types of cancers. The PPI enrichment analyses of the tumor and normal CoRePPI networks suggest that the co-regulating TFs and miRNAs are significantly associated with the specific biological processes, diseases and pathways. In addition, comparing with the two non-condition-specific approaches, the tumor CoRePPI network is found to have the most enriched cancer-related PPIs. Altogether, the results uncover the combined regulatory patterns of TFs and miRNAs on the PPIs, and may provide new insights for research in cancer-associated TFs and miRNAs.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Thuc Duy Le, Lin Liu, Jianfeng He, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        Gene
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/coreppi/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://linkinghub.elsevier.com/retrieve/pii/S0378-1119%2815%2901432-8">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/284888243_A_novel_framework_for_inferring_condition-specific_TF_and_miRNA_co-regulation_of_protein-protein_interactions">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/ensemble/" itemprop="url">Ensemble Methods for MiRNA Target Prediction from Expression Data</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Background: microRNAs (miRNAs) are short regulatory RNAs that are involved in several diseases, including cancers. Identifying miRNA functions is very important in understanding disease mechanisms and determining the efficacy of drugs. An increasing number of computational methods have been developed to explore miRNA functions by inferring the miRNA-mRNA regulatory relationships from data. Each of the methods is developed based on some assumptions and constraints, for instance, assuming linear relationships between variables. For such reasons, computational methods are often subject to the problem of inconsistent performance across different datasets. On the other hand, ensemble methods integrate the results from individual methods and have been proved to outperform each of their individual component methods in theory.Results: In this paper, we investigate the performance of some ensemble methods over the commonly used miRNA target prediction methods. We apply eight different popular miRNA target prediction methods to three cancer datasets, and compare their performance with the ensemble methods which integrate the results from each combination of the individual methods. The validation results using experimentally confirmed databases show that the results of the ensemble methods complement those obtained by the individual methods and the ensemble methods perform better than the individual methods across different datasets. The ensemble method, Pearson+IDA+Lasso, which combines methods in different approaches, including a correlation method, a causal inference method, and a regression method, is the best performed ensemble method in this study. Further analysis of the results of this ensemble method shows that the ensemble method can obtain more targets which could not be found by any of the single methods, and the discovered targets are more statistically significant and functionally enriched. The source codes, datasets, miRNA target predictions by all methods, and the ground truth for validation are available in the Supplementary materials.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Thuc Duy Le, Junpeng Zhang, Lin Liu, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        PLoS ONE
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/ensemble/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0131627">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/279447453_Ensemble_Methods_for_MiRNA_Target_Prediction_from_Expression_Data">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/mirna-tf/" itemprop="url">From miRNA regulation to miRNA-TF co-regulation: computational approaches and challenges</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        microRNAs (miRNAs) are important gene regulators. They control a wide range of biological processes and are involved in several types of cancers. Thus, exploring miRNA functions is important for diagnostics and therapeutics. To date, there are few feasible experimental techniques for discovering miRNA regulatory mechanisms. Alternatively, predictions of miRNA-mRNA regulatory relationships by computational methods have increasingly achieved promising results. Computational approaches are proving their ability as effective tools in reducing the number of biological experiments that must be conducted and to assist with the design of the experiments. In this review, we categorize and review different computational approaches to identify miRNA activities and functions, including the co-regulation of miRNAs and transcription factors. Our main focuses are on the recent approaches that use multiple data types for exploring miRNA functions. We discuss the remaining challenges in the evaluation and selection of models based on the results from a case study. Finally, we analyse the remaining challenges of each computational approach and suggest some future research directions.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Thuc Duy Le, Lin Liu, Junpeng Zhang, Bing Liu, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        Briefings in Bioinformatics
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirna-tf/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bib/article-lookup/doi/10.1093/bib/bbu023">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/263897723_From_miRNA_regulation_to_miRNA-TF_co-regulation_Computational_approaches_and_challenges">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/directtarget/">
        <img src="/img/DirectTarget.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/directtarget/" itemprop="url">Identifying direct miRNA-mRNA causal regulatory relationships in heterogeneous data</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Discovering the regulatory relationships between microRNAs (miRNAs) and mRNAs is an important problem that interests many biologists and medical researchers. A number of computational methods have been proposed to infer miRNA-mRNA regulatory relationships, and are mostly based on the statistical associations between miRNAs and mRNAs discovered in observational data. The miRNA-mRNA regulatory relationships identified by these methods can be both direct and indirect regulations. However, differentiating direct regulatory relationships from indirect ones is important for biologists in experimental designs. In this paper, we present a causal discovery based framework (called DirectTarget) to infer direct miRNA-mRNA causal regulatory relationships in heterogeneous data, including expression profiles of miRNAs and mRNAs, and miRNA target information. DirectTarget is applied to the Epithelial to Mesenchymal Transition (EMT) datasets. The validation by experimentally confirmed target databases suggests that the proposed method can effectively identify direct miRNA-mRNA regulatory relationships. To explore the upstream regulators of miRNA regulation, we further identify the causal feedforward patterns (CFFPs) of TF-miRNA-mRNA to provide insights into the miRNA regulation in EMT. DirectTarget has the potential to be applied to other datasets to elucidate the direct miRNA-mRNA causal regulatory relationships and to explore the regulatory patterns.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Thuc Duy Le, Lin Liu, Bing Liu, Gregory J Goodall, Jianfeng He, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        Journal of Biomedical Informatics
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/directtarget/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://www.sciencedirect.com/science/article/pii/S1532046414001865?via%3Dihub">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/265170848_Identifying_direct_miRNA-mRNA_causal_regulatory_relationships_in_heterogeneous_data">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/condition_specific_mirna_activity/">
        <img src="/img/condition_specific_miRNA_activity.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/condition_specific_mirna_activity/" itemprop="url">Inferring condition-specific miRNA activity from matched miRNA and mRNA expression data</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        MOTIVATION: MicroRNAs (miRNAs) play crucial roles in complex cellular networks by binding to the messenger RNAs (mRNAs) of protein coding genes. It has been found that miRNA regulation is often condition-specific. A number of computational approaches have been developed to identify miRNA activity specific to a condition of interest using gene expression data. However, most of the methods only use the data in a single condition, and thus, the activity discovered may not be unique to the condition of interest. Additionally, these methods are based on statistical associations between the gene expression levels of miRNAs and mRNAs, so they may not be able to reveal real gene regulatory relationships, which are causal relationships. RESULTS: We propose a novel method to infer condition-specific miRNA activity by considering (i) the difference between the regulatory behavior that an miRNA has in the condition of interest and its behavior in the other conditions; (ii) the causal semantics of miRNA-mRNA relationships. The method is applied to the epithelial-mesenchymal transition (EMT) and multi-class cancer (MCC) datasets. The validation by the results of transfection experiments shows that our approach is effective in discovering significant miRNA-mRNA interactions. Functional and pathway analysis and literature validation indicate that the identified active miRNAs are closely associated with the specific biological processes, diseases and pathways. More detailed analysis of the activity of the active miRNAs implies that some active miRNAs show different regulation types in different conditions, but some have the same regulation types and their activity only differs in different conditions in the strengths of regulation.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Thuc Duy Le, Lin Liu, Bing Liu, Gregory J Goodall, Jianfeng He, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        Bioinformatics
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/condition_specific_mirna_activity/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu489">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/264247884_Inferring_condition-specific_miRNA_activity_from_matched_miRNA_and_mRNA_expression_data">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://zhangjunpeng411.github.io/publication/fmrms/">
        <img src="/img/FMRM.png" class="pub-banner" itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://zhangjunpeng411.github.io/publication/fmrms/" itemprop="url">Inferring functional miRNA-mRNA regulatory modules in epithelial-mesenchymal transition with a probabilistic topic model</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        MicroRNAs (miRNAs) play important roles in gene regulatory networks. In this paper, we propose a probabilistic topic model to infer regulatory networks of miRNAs and their target mRNAs for specific biological conditions at the post-transcriptional level, so-called functional miRNA-mRNA regulatory modules (FMRMs). The probabilistic model used in this paper can effectively capture the relationship between miRNAs and mRNAs in specific cellular conditions. Furthermore, the proposed method identifies negatively and positively correlated miRNA-mRNA pairs which are associated with epithelial, mesenchymal, and other condition in EMT (epithelial-mesenchymal transition) data set, respectively. Results on EMT data sets show that the inferred FMRMs can potentially construct the biological chain of &lsquo;miRNA→mRNA→condition&rsquo; at the post-transcriptional level.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Junpeng Zhang, Bing Liu, Jianfeng He, Lei Ma, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        Computers in Biology and Medicine
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/fmrms/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://linkinghub.elsevier.com/retrieve/pii/S0010-4825%2811%2900251-4">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/216875030_Inferring_functional_miRNA-mRNA_regulatory_modules_in_epithelial-mesenchymal_transition_with_a_probabilistic_topic_modle">
  Custom Link
</a>


      </div>

    </div>
  </div>
</div>

    
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="publications" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent Publications</h1>
    
    
    <p class="view-all">
      <a href="/publication/">
        More Publications
        <i class="fa fa-angle-double-right"></i>
      </a>
    </p>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    <ul class="fa-ul">
      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Computational methods for identifying miRNA sponge interactions</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirna_sponge_review/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bib/article-abstract/18/4/577/2562759/Computational-methods-for-identifying-miRNA-sponge?redirectedFrom=fulltext">
  PDF
</a>






<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Inferring miRNA sponge co-regulation of protein-protein interactions in human breast cancer</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirscoppi/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1672-2">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/316784504_Inferring_miRNA_sponge_co-regulation_of_protein-protein_interactions_in_human_breast_cancer">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Identifying miRNA sponge modules using biclustering and regulatory scores</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirsm/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1467-5">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/315062626_Identifying_miRNA_sponge_modules_using_biclustering_and_regulatory_scores">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Identifying miRNA synergistic regulatory networks in heterogeneous human data via network motifs</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirsrn/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://pubs.rsc.org/-/content/articlelanding/2016/mb/c5mb00562k/unauth#!divAbstract">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/284765822_Identifying_miRNA_synergistic_regulatory_network_in_heterogeneous_human_data_via_network_motifs">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">miRLAB: An R Based Dry Lab for Exploring miRNA-mRNA Regulatory Relationships</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirlab/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0145386">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/288830678_MiRLAB_An_R_based_dry_lab_for_exploring_miRNA-mRNA_regulatory_relationships">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">A novel framework for inferring condition-specific TF and miRNA co-regulation of protein-protein interactions</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/coreppi/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://linkinghub.elsevier.com/retrieve/pii/S0378-1119%2815%2901432-8">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/284888243_A_novel_framework_for_inferring_condition-specific_TF_and_miRNA_co-regulation_of_protein-protein_interactions">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Ensemble Methods for MiRNA Target Prediction from Expression Data</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/ensemble/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0131627">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/279447453_Ensemble_Methods_for_MiRNA_Target_Prediction_from_Expression_Data">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">From miRNA regulation to miRNA-TF co-regulation: computational approaches and challenges</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/mirna-tf/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bib/article-lookup/doi/10.1093/bib/bbu023">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/263897723_From_miRNA_regulation_to_miRNA-TF_co-regulation_Computational_approaches_and_challenges">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Identifying direct miRNA-mRNA causal regulatory relationships in heterogeneous data</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/directtarget/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://www.sciencedirect.com/science/article/pii/S1532046414001865?via%3Dihub">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/265170848_Identifying_direct_miRNA-mRNA_causal_regulatory_relationships_in_heterogeneous_data">
  Custom Link
</a>

</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Inferring condition-specific miRNA activity from matched miRNA and mRNA expression data</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/publication/condition_specific_mirna_activity/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu489">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://zhangjunpeng411.github.io/project/bioinformatics/">
  Project
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.researchgate.net/publication/264247884_Inferring_condition-specific_miRNA_activity_from_matched_miRNA_and_mRNA_expression_data">
  Custom Link
</a>

</p>
</li>

      
    </ul>
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="talks" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent &amp; Upcoming Talks</h1>
    
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    <ul class="fa-ul">
      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://zhangjunpeng411.github.io/talk/apbc/">Identifying miRNA sponge modules using biclustering and regulatory scores</a></span>
  <p>
    1017-09-01,
    
            The Fifteenth Asia Pacific Bioinformatics Conference
    
  </p>
</li>

      
    </ul>
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="posts" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">

    <h1>Recent Posts</h1>
    
    

  </div>
  <div class="col-xs-12 col-md-8">

    

    
      
        
        


<div class="article-list-item" itemscope itemprop="blogPost">
  
  
  <h3 class="article-title" itemprop="name">
    <a href="https://zhangjunpeng411.github.io/post/hugo-demo/" itemprop="url">Getting started with the Academic framework for Hugo</a>
  </h3>
  

<div class="article-metadata">

  <span class="article-date">
    <time datetime="2017-08-23 00:00:00 &#43;0000 UTC" itemprop="datePublished">
      23087-09-08
    </time>
  </span>

  

  
  
  
  <span class="article-tags">
    <i class="fa fa-tags"></i>
    
    <a href="/tags/academic">academic</a
    >, 
    
    <a href="/tags/hugo">hugo</a
    >
    
  </span>
  
  

  

</div>

  <div class="article-style" itemprop="articleBody">
    
    <p>Create a beautifully simple personal or academic website in under 10 minutes.</p>
    
  </div>
  <p class="read-more">
    <a href="https://zhangjunpeng411.github.io/post/hugo-demo/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="projects" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Projects</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    <div class="project-toolbar">
      <div id="filters">
        <div class="btn-toolbar">
          <div class="btn-group">
          
            <a href="#" data-filter="*" class="btn btn-primary btn-large active">All</a>
          
            <a href="#" data-filter=".miRNA-sponge" class="btn btn-primary btn-large">Bioinformatics</a>
          
            <a href="#" data-filter=".demo" class="btn btn-primary btn-large">Other</a>
          
          </div>
        </div>
      </div>
    </div>

    

    <div id="container-projects" class="row isotope">

      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item demo">
        <div class="card">
          
          <div class="card-text">
            <h4><a href="http://example.org" target="_blank">External Project</a></h4>
            <div class="card-desription">
              <p>An example of linking directly to an external project website using <code>external_link</code>.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioinformatics">
        <div class="card">
          
          <div class="card-text">
            <h4><a href="https://zhangjunpeng411.github.io/project/mirna_sponge/" >miRNA sponge</a></h4>
            <div class="card-desription">
              <p>Study on the identification and functional characteristics of miRNA sponge causal networks and modules in cancer</p>
            </div>
          </div>
        </div>
      </div>
      

    </div>

    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="teaching" class="home-section">
    <div class="container">
      


<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Teaching</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <p>I am a teaching instructor for the following courses at Dali University:</p>

<ul>
<li>CS01: Fundamentals and Applications of Matlab</li>
<li>CS02: Biomedical Image Processing</li>
<li>CS03: Biomedical Statistical Analysis</li>
<li>CS04: Basics for Bioinformatics</li>
<li>CS05: Introduction to Data Mining</li>
<li>CS06: Course Design of Data Mining</li>
<li>CS07: Writing of Technological Thesis</li>
</ul>

  </div>
</div>

    </div>
  </section>

  
  
  <section id="contact" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Contact</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    <ul class="fa-ul" itemscope>

      
      <li>
        <i class="fa-li fa fa-envelope fa-2x" aria-hidden="true"></i>
        <span id="person-email" itemprop="email"><a href="mailto:zhangjunpeng_411@yahoo.com">zhangjunpeng_411@yahoo.com</a></span>
      </li>
      

      

      

      

      

      
      <li>
        <i class="fa-li fa fa-map-marker fa-2x" aria-hidden="true"></i>
        <span id="person-address" itemprop="address">Hongsheng Road #2, Gucheng Campus, Dali University, Yunnan, 671003, P. R. China</span>
      </li>
      

      
      <li>
        <i class="fa-li fa fa-clock-o fa-2x" aria-hidden="true"></i>
        <span>Monday to Friday or Email for appointment</span>
      </li>
      

    </ul>

  </div>
</div>

    </div>
  </section>



<footer class="site-footer">
   <div style="width:350px;margin:auto;">
   <script type="text/javascript" src="//rf.revolvermaps.com/0/0/1.js?i=523xdpqcnuk&amp;s=220&amp;m=0&amp;v=false&amp;r=false&amp;b=000000&amp;n=false&amp;c=ff0000" async="async"></script>  
   </div>
  
  <div class="container">
    <p class="powered-by">

      &copy; 2017 Junpeng Zhang &middot; 

      Powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha512-jGsMH83oKe9asCpkOVkBnUrDDTp8wl+adkB2D+//JtlxO4SrLoJdhbOysIFQJloQFD+C4Fl1rMsQZF76JjV0eQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.2/imagesloaded.pkgd.min.js" integrity="sha512-iHzEu7GbSc705hE2skyH6/AlTpOfBmkx7nUqTLGzPYR+C1tRaItbRlJ7hT/D3YQ9SV0fqLKzp4XY9wKulTBGTw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/TweenMax.min.js" integrity="sha512-Z5heTz36xTemt1TbtbfXtTq5lMfYnOkXM2/eWcTTiLU01+Sw4ku1i7vScDc8fWhrP2abz9GQzgKH5NGBLoYlAw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/plugins/ScrollToPlugin.min.js" integrity="sha512-CDeU7pRtkPX6XJtF/gcFWlEwyaX7mcAp5sO3VIu/ylsdR74wEw4wmBpD5yYTrmMAiAboi9thyBUr1vXRPA7t0Q==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

